ices/86053.rs

```diff
@@ -0,0 +1,2 @@
+fn ordering4 < 'a , 'b     > ( a :            ,   self , self ,   self ,   self , ... ,   self ,   self , ... ) where F : FnOnce ( & 'a & 'b usize ) {
```

Comment:
The formatting doesn't have meaning here, we can use https://github.com/rust-lang/rust/issues/86053#issuecomment-855672258 to simplify.

Reply:
Resolved in 59274cb.